src/renderer/src/chat.ts

```diff
@@ -1,67 +1,98 @@
-import { ChatCompletionRequestMessage, ChatCompletionRequestMessageRoleEnum, Configuration, OpenAIApi } from "openai";
-import { Chitchat } from "../../shared/storage";
-import { FAKE_CHITCHAT } from "../flags";
-import dedent from "dedent";
+import { ChatCompletionRequestMessage, ChatCompletionRequestMessageRoleEnum } from "openai";
+import { getClient } from "./openai";
 
-let executeChitChat: (chitchat: Chitchat, query: string) => Promise<string>;
+let instance: ChatView
 
-executeChitChat = async (chitchat: Chitchat, query: string): Promise<string> => {
-  const config = new Configuration({
-    apiKey: window.storeGet("openAiAPIKey"),
-  })
-
-  // Electron doesn't like the User-Agent header being set
-  delete config.baseOptions.headers["User-Agent"];
+// Model enum
+export enum Model {
+  Gpt35Turbo = "gpt-3.5-turbo",
```

Comment:
We have a ModelName type in storage.ts. Can we unify these?

Reply:
done